SensusUI/PromptForInputsPage.cs

```diff
@@ -121,36 +142,49 @@ public PromptForInputsPage(InputGroup inputGroup, int stepNumber, int totalSteps
             if (_displayedInputCount > 0)
                 contentLayout.Children.Add(new BoxView { Color = Color.Transparent, HeightRequest = inputSeparatorHeight });
 
-            if (anyRequired)
-                contentLayout.Children.Add(new Label
-                    {
-                        Text = "* Required Field",
-                        FontSize = 15,
-                        TextColor = Color.Red,
-                        HorizontalOptions = LayoutOptions.FillAndExpand
-                    });
-
             StackLayout navigationStack = new StackLayout
             {
-                Orientation = StackOrientation.Horizontal,
+                Orientation = StackOrientation.Vertical,
                 HorizontalOptions = LayoutOptions.FillAndExpand
             };
 
+            StackLayout previousNextStack = new StackLayout
+                {
+                    Orientation = StackOrientation.Horizontal,
+                    HorizontalOptions = LayoutOptions.FillAndExpand,
+//                    Padding = new Thickness(0, 0, 0, 15)
+                };
+
+            StackLayout previousStack = new StackLayout
+                {
+                    Orientation = StackOrientation.Horizontal,
+                    HorizontalOptions = LayoutOptions.FillAndExpand
+                };
+
+            StackLayout nextStack = new StackLayout
+                {
+                    Orientation = StackOrientation.Horizontal,
+                    HorizontalOptions = LayoutOptions.FillAndExpand
+                };
+
+            previousNextStack.Children.Add(previousStack);
+            previousNextStack.Children.Add(nextStack);
+
```

Comment:
I've been thinking about what I suggested, and maybe it doesn't make sense to display a previous button in every case, especially since PromptForInputsPage is used for protocol consent too. I could change this so only a centered next/submit is displayed when on the first page. What do you think?